green/output.py

```diff
@@ -144,14 +145,16 @@ class GreenStream(object):
 
     def __init__(self, stream, override_appveyor=False, disable_windows=False):
         self.stream = stream
+        self.disable_windows = disable_windows
+        self.override_appveyor = override_appveyor
         # Ironically, AppVeyor doesn't support windows win32 system calls for
         # colors, but it WILL interpret posix ansi escape codes!
-        on_windows = platform.system() == 'Windows'
-        on_appveyor = os.environ.get('APPVEYOR', False)
+        self.on_windows = platform.system() == 'Windows'
+        self.on_appveyor = os.environ.get('APPVEYOR', False)
```

Comment:
likewise, I expect this should be a package level constant.  I dont see how this env var could ever be expected to change during an execution of `green`.

Also, wherever it is placed, it should be a private attribute, as this is not something that should become part of the public `green` contract.